docs/api/users.md

```diff
@@ -51,3 +51,47 @@ Content-Type: application/json
   "updated": "2016-12-22T02:14:05Z"
 }
 ```
+
+
+## List users
+
+### URL
+
+```http
+GET /users HTTP/1.1
+```
```

Comment:
This currently lists users not in any organisation (admins). But when implemented, it will list users based on the current scope. (All users in an organisation if an organisational user is active, else admins.)